custom-widget-builder/src/HtmlTemplatesGenerator.ts

```diff
@@ -190,7 +190,9 @@ export class HtmlTemplatesGenerator {
       }
       let attributeName = CustomWidgetBuilder.camelToKebabCase(prop.name);
       if (type === FwkType.Angular && prop.bond === Bond.Variable) {
-        props += `[(ngModel)]="properties.${prop.name}"\n\t`;
+        props += `[(${prop.name})]="properties.${prop.name}"\n\t`;
+      } else if (type === FwkType.Angular && prop.bond != Bond.Constant) {
```

Comment:
What happens if property Bond is Variable ? We will enter in first and second condition right and override the return value.

Missing prop.bond !== Bond.Variable in this condition i think.

Reply:
If Angular and bond is Variable, we will enter only in the first condition, since we have 'else if' for the second,  don't we ?

Reply:
Yeah, you're right, sorry I don't know where I am when I do these comments :roll_eyes: